lib/clamp-link.in

```diff
@@ -231,7 +231,7 @@ do
     else
       # this is .a static library file specified at the commad line
       if [ -f $ARG ]; then
-        DETECTED_STATIC_LIBRARY=$ARG
+        DETECTED_STATIC_LIBRARY=$(readlink -f $ARG)
```

Comment:
@scchan I've also fixed the same spot in a commit just now. :) I'd like to retain the unit test though. Could you alter this pull request so it only contains the test? Thanks.